src/Microsoft.VisualStudio.ProjectSystem.FSharp.VS/ProjectSystem/VS/Imaging/FSharpProjectImageProvider.cs

```diff
@@ -11,7 +11,6 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.Imaging
     /// </summary>
     [Export(typeof(IProjectImageProvider))]
     [AppliesTo(ProjectCapability.FSharp)]
-    [Order(2)]
```

Comment:
@srivatsn I couldn't see how this was applied - it appears deliberate. Why does it need it? I can't expect more than one of these in place at a time.